drivers/interrupt_controller/Kconfig.multilevel

```diff
@@ -19,7 +19,7 @@ config MULTI_LEVEL_INTERRUPTS
 	  controller".)
 
 if MULTI_LEVEL_INTERRUPTS
-config 1ST_LEVEL_INTERRUPT_BITS
+config LEVEL_1_INTERRUPT_BITS
```

Comment:
Remember, Kconfig is API, so you need to add the new option, make the new option default to the old option, and mark the old option as deprecated.

Deprecated API components can be removed after 2 minor releases.

Hard to say if it's worth the fuss though - it might be better to simply create a new chained interrupt architecture.

Reply:
Probably easier to workaround this with another macro..